client/src/components/payments/stripeStyle.js

```diff
@@ -2,6 +2,68 @@
 //of payment card. If you figure out how to use MUI
 //components for these items please let me know. -taryn
 
+const checkoutStyles = (muiBaseTheme) => ({
+    card: {
+        width: '500px',
+        margin: '50px auto',
+        padding: muiBaseTheme.spacing.unit * 3,
+        transition: '0.5s',
+        boxShadow: '0 8px 40px -12px rgba(0,0,0,0.3)',
+        '&:hover': {
+            boxShadow: '0 16px 70px -12.125px rgba(0,0,0,0.3)'
+        },
+        [muiBaseTheme.breakpoints.down('sm')]: {
+            width: '100%',
+            padding: "0"
+        }
+    },
+    content: {
+        textAlign: 'center',
+        padding: muiBaseTheme.spacing.unit
```

Comment:
just as an fyi, I think material-ui is deprecating this or something. got a warning on console about it on other pages

Reply:
yeah I saw that. what have you been using instead?